ajax_with_jquery_exercise/hackernews2.js

```diff
@@ -0,0 +1,140 @@
+
+
+$(function(){
```

Comment:
Good start! Try to get the favoriting working when you can.  At least the creating of a favorite.